barbot/bars.py

```diff
@@ -110,6 +110,8 @@ def match_bars(self, searches: Sequence[str]) -> Tuple[List[str], List[Bar]]:
 
 class TestBars(unittest.TestCase):
     def test_normalize_name(self):
```

Comment:
nitpick: each test should only be asserting a single aspect, I think it would be good to move these new assertions into a new test, possibly named 'test normalization preserves numbers' (thanks for addings tests by the way 🙇)

Reply:
consider it done https://github.com/ceresgalax/barbot/pull/8